eval-config.nix

```diff
@@ -16,6 +16,7 @@ let
   };
 
   eval = lib.evalModules (builtins.removeAttrs args [ "lib" ] // {
+    class = "darwin";
```

Comment:
> `This would be remove in the future, Prefer _module.args option instead.`

From https://github.com/NixOS/nixpkgs/blob/6cee3b5893090b0f5f0a06b4cf42ca4e60e5d222/lib/modules.nix#L84

Would it be better to add it into the `argsModule` above then?

Reply:
I think that comment is misplaced and referring to the `args` argument. I don't see how you could set `class` through `_module.args`, from the rest of the code.